src/api/spec/support/shared_examples/features/beta/user_tab.rb

```diff
@@ -40,7 +40,7 @@
       expect(page).to have_selector("a.remove-user[data-object='user_tab_user']")
     end
 
-    scenario 'Add non existent user' do
```

Comment:
Which cop is marking this as an offense? It would be helpful to include this in the commit message or description.